hera_qm/xrfi.py

```diff
@@ -70,14 +70,32 @@ def flag_xants(uv, xants, inplace=True):
         return uvo
 
 
+def resolve_xrfi_path(xrfi_path, file):
+    """ Determine xrfi_path based on given directory or default to dirname of given file.
+    Args:
+        xrfi_path (str): Directory to write xrfi outputs.
+        file (str): Filename to determine backup directory if xrfi_path == ''
+    Returns:
+        dirname (str): If xrfi_path != '', returns xrfi_path. Otherwise returns
+            directory of file.
+    """
+    if xrfi_path != '':
```

Comment:
Would it make sense to make this more general? Perhaps check whether the directory exists?

Reply:
added that functionality.